pkg/controller/certificates/requestmanager/requestmanager_controller.go

```diff
@@ -332,10 +332,13 @@ func (c *controller) createNewCertificateRequest(ctx context.Context, crt *cmapi
 			IssuerRef: crt.Spec.IssuerRef,
 			Request:   csrPEM.Bytes(),
 			IsCA:      crt.Spec.IsCA,
-			Usages:    crt.Spec.Usages,
 		},
 	}
 
+	if crt.Spec.EncodeUsagesInRequest == nil || *crt.Spec.EncodeUsagesInRequest {
```

Comment:
This part is incorrect, the usages in the CertificateRequest should still be present, they can be left empty but that will default them.

https://github.com/jetstack/cert-manager/pull/3211/files this is where we added the usages into the CSR body